src/flash/core/utilities/flash_cli.py

```diff
@@ -107,6 +105,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def get_class_signature_functions(classes):
```

Comment:
This was a jsonargparse internal function that no longer exists, which is why I copied it here.